st2common/st2common/models/db/execution.py

```diff
@@ -71,6 +71,7 @@ class ActionExecutionDB(stormbase.StormFoundationDB):
     parent = me.StringField()
     children = me.ListField(field=me.StringField())
     log = me.ListField(field=me.DictField())
+    delay = me.IntField()
```

Comment:
We default it to ``0`` on another model below so we should also do the same here.